invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/subjects.html

```diff
@@ -6,13 +6,31 @@
   it under the terms of the MIT License; see LICENSE file for more details.
 #}
 
-{% set subjects = record.ui.subjects %}
+{% set subjects = record.metadata.subjects %}
 
 {% if subjects %}
-<dt><b>{{_('Subjects')}}</b></dt>
-<dd class="top-bottom-padded">
-  {%- for subject in subjects %}
-  <span class="label label-keyword label-aligned">{{ subject.title_l10n}}</span>
+<dd class="top-bottom-padded" id="record-subjects">
+  {%- for scheme, subjects in subjects|selectattr("scheme")|groupby('scheme') %}
```

Comment:
minor just to distinguish between the many subjects around :rabbit2: 

```suggestion
  {%- for scheme, subjects in all_subjects|selectattr("scheme")|groupby('scheme') %}
```
(and change for `all_subjects` above (or something similar)